httpclient/tests/httpclient_test.rs

```diff
@@ -43,7 +43,7 @@ async fn get_request(_opt: &TestOptions) -> RpcResult<()> {
     let ctx = Context::default();
 
     let resp = client
-        .request(&ctx, &HttpRequest::get("https://wttr.in/London?format=3"))
+        .request(&ctx, &HttpRequest::get("https://wttr.in/London"))
```

Comment:
Does this break the test?

Reply:
I made that change because it started failing with the format arg on my machine but succeeded without the format arg Worked on my machine, but maybe there's some other problem with the site.  I replaced wttr.in with https://1.1.1.1 which should be faster and more reliable.